File: parentControlApplication/src/main/java/ru/nekit/android/qls/parentControl/setupWizard/steps/StartSetupWizardFragment.java
package ru.nekit.android.qls.parentControl.setupWizard.steps;

import android.support.annotation.LayoutRes;
import android.support.annotation.NonNull;
import android.view.View;

import ru.nekit.android.qls.parentControl.R;
import ru.nekit.android.qls.parentControl.setupWizard.ParentControlSetupWizardFragment;

public class StartSetupWizardFragment extends ParentControlSetupWizardFragment {

    public static StartSetupWizardFragment getInstance() {
        return new StartSetupWizardFragment();
    }

    @LayoutRes
    @Override
    protected int getLayoutId() {
        return R.layout.sw_setup_start;
    }

    @Override
    protected void onSetupStart(@NonNull View view) {
        setNextButtonText(getSetupWizard().setupIsStart() ?
                R.string.label_setup_wizard_continue : R.string.label_start_setup_wizard);
    }
}

File: questPresentation/src/main/java/ru/nekit/android/qls/setupWizard/view/StartSetupWizardFragment.kt
package ru.nekit.android.qls.setupWizard.view

import android.support.annotation.LayoutRes
import android.view.View
import ru.nekit.android.qls.R
import ru.nekit.android.qls.R.string.label_setup_wizard_continue
import ru.nekit.android.qls.R.string.label_start_setup_wizard
import ru.nekit.android.utils.ParameterlessSingletonHolder

class StartSetupWizardFragment : QuestSetupWizardFragment() {

    @LayoutRes
    override fun getLayoutId() = R.layout.sw_setup_start

    override fun onSetupStart(view: View) {
        setupWizard.setupIsStart {
            setNextButtonText(if (it)
                label_setup_wizard_continue
            else
                label_start_setup_wizard)
        }
    }

    companion object : ParameterlessSingletonHolder<StartSetupWizardFragment>(::StartSetupWizardFragment)
}
